### PR TITLE
Rich text: run e2e tests in Firefox and Webkit

### PR DESCRIPTION
**Pull request by ellatrix**
## Why?

This is absolutely necessary when doing an refactors of rich text. Currently Firefox is manual testing.

### CODE REVIEW

File: packages/e2e-test-utils-playwright/src/page-utils/press-keys.ts
Comment (kevin940726) [inline packages/e2e-test-utils-playwright/src/page-utils/press-keys.ts:1]: Why this change is needed and what problem does it solve? Might be worth noting that there's currently an open PR in https://github.com/WordPress/gutenberg/pull/55030 that drastically refactors this util.

Comment (ellatrix) [reply packages/e2e-test-utils-playwright/src/page-utils/press-keys.ts:1]: See my comment here: https://github.com/WordPress/gutenberg/pull/53482/files#r1288958127

The emulator just doesn't work at all in FF.
